src/dag-link/index.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const mh = require('multihashes')
+const assert = require('assert')
+
+// Link represents an IPFS Merkle DAG Link between Nodes.
+class DAGLink {
+  constructor (name, size, multihash) {
```

Comment:
But that is not handled properly as `new DAGLink(123, 'myhash')` throws at the moment